cli/lib/kontena/cli/helpers/log_helper.rb

```diff
@@ -3,8 +3,8 @@ module LogHelper
 
     def self.included(base)
       if base.respond_to?(:option)
-        base.option ["-t", "--tail"], :flag, "Tail (follow) logs", default: false
-        base.option "--lines", "LINES", "Number of lines to show from the end of the logs", default: 100 do |s|
+        base.option ["-f", "--follow"], :flag, "Follow log output", :attribute_name => :tail, default: false
+        base.option "--tail", "LINES", "Number of lines to show from the end of the logs", :attribute_name => :lines, default: 100 do |s|
```

Comment:
Change to `['--lines', '--tail']` to keep the old `--lines` as an alternative to --tail.